src/decode.hpp

```diff
@@ -194,6 +195,26 @@ amrl_msgs::Localization2DMsg decode(const void* const data) {
   return dst;
 }
 
+template <>
+geometry_msgs::PoseStamped decode(const void* const data) {
+  const fb::geometry_msgs::PoseStamped* src =
+      flatbuffers::GetRoot<fb::geometry_msgs::PoseStamped>(data);
+  geometry_msgs::PoseStamped dst;
+  
+  decode_header(src, dst);
+
+  dst.pose.orientation.x = src->pose()->orientation()->x();
+  dst.pose.orientation.y = src->pose()->orientation()->y();
+  dst.pose.orientation.z = src->pose()->orientation()->z();
+  dst.pose.orientation.w = src->pose()->orientation()->w();
+  dst.pose.position.x = src->pose()->position()->x();
+  dst.pose.position.y = src->pose()->position()->y();
+  dst.pose.position.z = src->pose()->position()->z();
```

Comment:
we ought to do this by implementing `decode<Point3D>` and `decode<Quaternion>` and using the `decode_obj()` helper ([example](https://github.com/ut-amrl/robofleet_client/blob/6279ec21988761cbb8a6fb1c618324e008c4afb2/src/decode.hpp#L140)), but this is fine for now since we don't have those implemented.